deploy/testnet/scripts/init-validator.sh

```diff
@@ -38,15 +36,9 @@ then
     ONOMY_ORCHESTRATOR_NAME=$(jq -r .name $ONOMY_HOME/orchestrator_key.json)
 fi
 
-while [[ $ONOMY_ORCHESTRATOR_NAME == '' ]]
-do
-   # The name of onomy orchestrator
-  read -r -p "Enter a name for your orchestrator [orchestrator]:" ONOMY_ORCHESTRATOR_NAME
-  ONOMY_ORCHESTRATOR_NAME=${ONOMY_ORCHESTRATOR_NAME:-orchestrator}
-  $ONOMY keys add $ONOMY_ORCHESTRATOR_NAME $ONOMY_KEYRING_FLAG --output json | jq . >> $ONOMY_HOME/orchestrator_key.json
-done
-
-
+read -r -p "Enter a name for your orchestrator [orchestrator]:" ONOMY_ORCHESTRATOR_NAME
+ONOMY_ORCHESTRATOR_NAME=${ONOMY_ORCHESTRATOR_NAME:-orchestrator}
+onomyd keys add $ONOMY_ORCHESTRATOR_NAME $ONOMY_KEYRING_FLAG --output json | jq . >> $ONOMY_HOME/orchestrator_key.json
```

Comment:
Line 55 & Line 59
We should format output and highlight the address part to make it stand out
```
TEXTFG=$(tput setaf 2)
BOLD=$(tput bold)
UNDERLINE=$(tput smul)
NORM=$(tput sgr0)


echo -e "Your validator address is ${TEXTFG}${BOLD}${UNDERLINE}$ONOMY_VALIDATOR_ADDRESS${NORM}"
echo -e "Your orchestrator address is ${TEXTFG}${BOLD}${UNDERLINE}$ONOMY_ORCHESTRATOR_ADDRESS${NORM}"
```